runner/src/internal.rs

```diff
@@ -69,8 +69,12 @@ pub struct RunExamples {
     pub build_server: BuildServer,
     #[structopt(long, help = "run server [default: true]")]
     pub run_server: Option<bool>,
-    #[structopt(long, help = "run clients [default: true]")]
-    pub run_clients: Option<bool>,
+    #[structopt(
+        long,
+        help = "client variant: [all, rust, cpp, go, nodejs, none] [default: all]",
```

Comment:
Just curious... since `all` is an option here, does it also make sense to support any subset of the clients? I think that could be done if the `--client-variant` flag takes a list of the clients.

Reply:
Will add in a separate PR